Makefile

```diff
@@ -26,13 +26,15 @@ DEVICES = $(BUILDDIR)/devices/dspic33e.o \
 a10: CFLAGS += -DBOARD_A10
 raspberrypi: CFLAGS += -DBOARD_RPI
 raspberrypi2: CFLAGS += -DBOARD_RPI2
+rpi4b: CFLAGS += -DBOARD_RPI4B
```

Comment:
I'd suggest _raspberrypi4b_ instead of _rpi4b_, to keep the same naming scheme as before